integration_tests/e2e/activities/attendance/multipleActivities.cy.ts

```diff
@@ -210,4 +211,64 @@ context('Record attendance', () => {
       'View or Edit',
     )
   })
+
+  it('Mark attendances for single activity when not in standalone mode', () => {
```

Comment:
Test to validate when only one activity session is selected for marking attendance